lib/parse.js

```diff
@@ -87,6 +87,10 @@ var parseValues = function parseQueryStringValues(str, options) {
             val = val.split(',');
         }
 
+        if (part.indexOf('[]=') > -1) {
+            val = utils.isArray(val) ? [val] : val;
```

Comment:
a perhaps simpler alternative here:
```suggestion
            val = [].concat([val]);
```

Reply:
@ljharb 

`val = [].concat([val]);`

will fail test `allows disabling array parsing` in the following case:

```
        var emptyBrackets = qs.parse('a[]=b', { parseArrays: false });
        st.deepEqual(emptyBrackets, { a: { 0: 'b' } });
```
expected: |-
      { a: { 0: 'b' } }
    actual: |-
      { a: { 0: [ 'b' ] } }

Reply:
ah, right, `[].concat([val])` is actually just `[val]`, so that's not the same.